storage/client_test.go

```diff
@@ -161,6 +164,7 @@ func (m *multiTestContext) addStore(t *testing.T) {
 	if err := store.Start(); err != nil {
 		t.Fatal(err)
 	}
+	eng.Start() // Add an extra reference count to the engine
```

Comment:
Explain in a comment that we're doing this so that the engine is not destroyed when we stop and restart the node.